grammars/javascript.cson

```diff
@@ -335,10 +335,18 @@
     'name': 'storage.type.js'
   }
   {
-    'match': '(?<!\\.)\\b(const|export|extends|implements|let|private|protected|public|static|var)(?!\\s*:)\\b'
+    'match': '(?<!\\.)\\b(export|extends|implements|let|private|protected|public|static|var)(?!\\s*:)\\b'
     'name': 'storage.modifier.js'
   }
   {
+    'match': '(?<!\\.)\\b(const)(?!\\s*:)\\b\\s+(\\w+)'
```

Comment:
I don't think `\w` is sufficient, it would match things like `const 5 = true;` and also not match `const $FOO = 3;`, I think it might need to be something like `([$_a-zA-Z][[$_a-zA-Z0-9]*)` instead.